### ISSUE TITLE
script is using my wave 3 command on wave 2 now.

### ISSUE THREAD

**Issue by w-san** [29988122/Fate-Grand-Order_Lua#272]
gh6,#,5,#,ad1efx21d1ef1j4

here is my string i'm using Atalanta, Arash, Nitocris as start.

Sometimes after Arash NP and the mobs don't die on battle 2.

It starts running my battle 3 commands.

I know the autoskill is kinda buggy.

Running JP on Nox.

Using Trail Ankulua 8.5.3

**Comment by TryBane** [comment]
Try using Ankulua 8.0

**Comment by w-san** [comment]
ok i'll give that a go

**Comment by TryBane** [comment]
Small lags can cause a miss in a tap. There is a way to fix it by providing a wait between skills which I did in the pending Preload-NP Pull Request, but that hasn't been approved for merge yet. As for the 8.0 trial version, [Try This](https://drive.google.com/uc?export=download&id=182bH3Y80aekWpN7D7e2SqzKfTiQrpiO8)

**Comment by w-san** [comment]
ahh the file says 8.5.0 is that the version you wanted me to try?

**Comment by TryBane** [comment]
Yes, my bad. I was seeing your 8.5.3, and trying to suggest 8.5.0, but instead suggested 8.0 XD

Sorry about that, but yes give 8.5 a try.

**Comment by w-san** [comment]
ahh XD no worries.

This is odd my phone keep saying the package is corrupted.
